### DISCUSSION TITLE
lotus chain data fill up the storage very fast

### DISCUSSION THREAD

**Discussion by potato1992** [filecoin-project/lotus#10743] [Storage Provider] [answered]
The lotus chain data seems to be growing really fast now. It  takes only 10 days to fill up a 2TB SSD. Anyone know the method of reducing the disk use?

Also, can the dev team verify they are only storing essential stuffs?

**Comment by segunjkf** [comment]
I also have the same issue filling up 1 TB in 24 hours.

**Comment by TippyFlitsUK** [comment, marked as answer]
This issue has been resolved with network version 19. Please le us know if it arises again 🙏
